Gemfile

```diff
@@ -15,6 +15,6 @@ end
 
 group :development do
   gem 'appraisal', git: 'https://github.com/thoughtbot/appraisal.git'
-  gem 'pry-byebug', '3.9.0'
+  gem 'pry-byebug'
```

Comment:
Needed to unpin pry-byebug to let it find the right version for each Ruby version